yarp.i

```diff
@@ -45,7 +45,7 @@
   // Try to translate std::string and std::vector to native equivalents
   %include "std_string.i"
   %typemaps_std_string(yarp::os::ConstString, char, SWIG_AsCharPtrAndSize,
-		       SWIG_FromCharPtrAndSize, %checkcode(STDSTRING));
```

Comment:
This file is temporary copy-pasted here from https://github.com/robotology/yarp/blob/master/bindings/yarp.i , so for the sake of porting changes back to the original file it would be easier to avoid unnecessary whitespace changes (by the way, I think we can put matlab-specific `#if`s directly in the original repo and remove this copy).